backend/src/graphql/resolvers/UserResolver.ts

```diff
@@ -86,6 +88,17 @@ export class UserResolver {
     if (!result.success) {
       throw new Error(result.data)
     }
+    const qluser = new User(result.data.user)
+    const user = new DbUser()
+    user.pubkey = Buffer.from(fromHex(qluser.pubkey))
```

Comment:
Why do we need fromHex here? The pubkey is a buffer.

Reply:
The Community Server db save the public key in binary format, the graphQL user in hex format.
Binary needs half the space of the hex version, but the hex version is easier for transport via http and debug.